spec/support/shared_vcr_setup.rb

```diff
@@ -36,7 +36,7 @@
     /codeclimate.com/ ===  URI(request.uri).host
   end
 
-  config.ignore_request { |request| URI(request.uri).port == 9200 } #Elasticsearch
+  config.ignore_request { |request| URI(request.uri).port == 9256 } #Elasticsearch
```

Comment:
This should probably match the range 9200-9299, so that we are ignoring requests for multiple Elasticsearch ports. Using a range rather than an array of a couple of ports gives us the flexibility to tweak the ports for future ES versions without changing the VCR setup each time.

Reply:
good point 👍